src/Take.Blip.Builder/Hosting/IConfiguration.cs

```diff
@@ -27,5 +27,6 @@ public interface IConfiguration
         string SqlStorageDriverTypeName { get; }
 
         string ContextType { get; }
-    }    
+        TimeSpan CacheContactExpiration { get; }
```

Comment:
Rename and add a blank line

```suggestion

        TimeSpan ContactCacheExpiration { get; }
```